test/hotspot/jtreg/compiler/c2/TestVerifyIterativeGVN.java

```diff
@@ -25,9 +25,9 @@
  * @test
  * @bug 8238756
  * @requires vm.debug == true & vm.flavor == "server"
- * @summary Run with -Xcomp to test -XX:+VerifyIterativeGVN in debug builds.
+ * @summary Run with -Xcomp to test -XX:VerifyIterativeGVN=1 in debug builds.
  *
- * @run main/othervm/timeout=300 -Xbatch -Xcomp -XX:+VerifyIterativeGVN compiler.c2.TestVerifyIterativeGVN
+ * @run main/othervm/timeout=300 -Xbatch -Xcomp -XX:VerifyIterativeGVN=1 compiler.c2.TestVerifyIterativeGVN
```

Comment:
As this is a Hello World sanity test for `VerifyIterativeGVN`, do we want to run this with both verifications on (i.e. `-XX:VerifyIterativeGVN=11`)?

Reply:
:+1: